src/index.html

```diff
@@ -4,19 +4,107 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>MOYO header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="header__img"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo header logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li
+            data-qa="hover"
+            class="header__item"
+          >
+            <a
+              href="#"
+              class="header__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
```

Comment:
There should be empty lines between sibling `li` elements for better readability, as per the checklist. Add an empty line between the `li` elements to create visual separation and improve readability.